src/main/java/net/sf/jabref/gui/maintable/MainTableColumn.java

```diff
@@ -144,4 +132,26 @@ public JLabel getHeaderLabel() {
             return new JLabel(getDisplayName());
         }
     }
+
+    public boolean isResolved(BibEntry entry) {
+        if (bibtexFields.isEmpty()) {
+            return false;
+        }
+
+        Optional<String> content = Optional.empty();
+        Optional<String> entryContent = Optional.empty();
+        for (String field : bibtexFields) {
+            if (BibEntry.TYPE_HEADER.equals(field) || "bibtextype".equals(field)) {
+                return false;
+            } else {
+                entryContent = entry.getFieldOptional(field);
+                content = BibDatabase.getResolvedField(field, entry, database.orElse(null));
```

Comment:
I didn't really understand what I commented on. You suggested naming still makes sense, but the method returns false if the content of the field is the same as the resolved content of the field, i.e., no string, crossrefs, or alias fields are used.